consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeBlockCreator.java

```diff
@@ -71,7 +69,7 @@ public MergeBlockCreator(
         protocolContext,
         protocolSchedule,
         minTransactionGasPrice,
-        minBlockOccupancyRatio,
+        1.0,
```

Comment:
Might be better if we use a constant here like we did in the test

Reply:
done, and also added a javadoc to explain why